tests/webview/ui/test_home.py

```diff
@@ -15,26 +15,61 @@
 
 @markers.webview
 @markers.nondestructive
-def test_splash_banner_loads(webview_base_url, selenium):
-    # GIVEN the main website URL and the Selenium driver
+@markers.parametrize('width,height', [(1024, 768), (640, 480)])
```

Comment:
We could also add this to the selenium fixture which would run all the tests with both resolutions.

Reply:
```python
@pytest.fixture(scope='function',
                params=[(1080, 1920), (414, 738)],
                ids=['Resolution: 1080x1920', 'Resolution: 414x738'])
def selenium(selenium, request):
    """Fixture to set custom selenium parameters.
    This fixture will also parametrize all of the tests to run them on both a
    Desktop resolution and a mobile resolution.
    Desktop size: 1920x1080
    Mobile size: 738x414 (iPhone 7+)
    """
    selenium.set_window_size(*request.param)
    return selenium
```

Reply:
We can hold off on this for now. It will make things take longer and I don't think the expectations of webview are high for responsiveness. We can pick out select tests like the hamburger nav to test and that's good 👍